client/src/curriculum/about.tsx

```diff
@@ -0,0 +1,26 @@
+import { HydrationData } from "../../../libs/types/hydration";
+import { CurriculumDoc, CurriculumData } from "../../../libs/types/curriculum";
+import { topic2css, useCurriculumDoc } from "./utils";
+import { RenderCurriculumBody } from "./body";
+import { CurriculumLayout } from "./layout";
+
+import "./index.scss";
+import "./about.scss";
+
+export function CurriculumAbout(props: HydrationData<any, CurriculumDoc>) {
+  const doc = useCurriculumDoc(props as CurriculumData);
```

Comment:
Don't like this assertion - if they types are incompatible due to HydrationData weirdness (which will take a lot to unpick), why not just take a CurriculumDoc as the argument in useCurriculumDoc?